.travis.yml

```diff
@@ -6,7 +6,7 @@ addons:
     - python-scipy
 install:
   - pip install flake8  # Not yet available in Ubuntu precise (https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise)
-  - pip install -r requirements.txt
```

Comment:
About `requirements.txt` you're right to remove it, since it's been packaged in PyPI for a long time now. We don't want to install `OpenFisca-Core` from git anymore.